br/pkg/restore/split.go

```diff
@@ -188,26 +181,44 @@ func (rs *RegionSplitter) hasHealthyRegion(ctx context.Context, regionID uint64)
 	return len(regionInfo.PendingPeers) == 0, nil
 }
 
-func (rs *RegionSplitter) isScatterRegionFinished(ctx context.Context, regionID uint64) (bool, error) {
+// isScatterRegionFinished check the latest successful operator and return the follow status:
+//
+//	return (finished, needRescatter, error)
+//
+// if the latest operator is not `scatter-operator`, or its status is SUCCESS, it's likely that the
```

Comment:
if the latest operator is not `scatter-operator`
---

Will it become a mistake when the scatter-operator canceller and another operator succeed?

Reply:
yes. But it is ignored usually.